pkg/scheduler/server/server.go

```diff
@@ -71,30 +76,46 @@ func New(opts Options) *Server {
 	apiLevel.Store(config.ActorAPILevel)
 
 	if opts.PlacementAddress != "" {
-		actorOptions := config.ActorsProviderOptions{
-			Config: config.Config{
-				ActorsService:    "placement:" + opts.PlacementAddress,
-				AppID:            opts.AppID,
-				HostAddress:      opts.HostAddress,
-				Port:             s.port,
-				PodName:          os.Getenv("POD_NAME"),
-				HostedActorTypes: config.NewHostedActors([]string{}),
+		// Create gRPC manager
+		grpcAppChannelConfig := &manager.AppChannelConfig{}
+		s.grpcManager = manager.NewManager(opts.Security, opts.Mode, grpcAppChannelConfig)
+		s.grpcManager.StartCollector()
+
+		act, _ := actors.NewActors(actors.ActorsOpts{
+			AppChannel:       nil,
+			GRPCConnectionFn: s.grpcManager.GetGRPCConnection,
+			Config: actors.Config{
+				Config: config.Config{
+					ActorsService:                 "placement:" + opts.PlacementAddress,
```

Comment:
Should these vars be added to the Scheduler `options.go`? I don't think we have all of these vars defined there: `opts.PlacementAddress`,`opts.AppID`, etc

Reply:
We have enough right now from Scheduler's options. This config is doing far more than we need.